### PR TITLE
docs(rollup): improve clarity

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

File: packages/rollup/install.md [hunk 1]
```diff
@@ -122,5 +124,33 @@ You could use this value to perform extra production-only optimizations.
 For example you could define a constant for enabling Debug:
 
 ```javascript
-const DEBUG = process.env['COMPILATION_MODE'] === 'dbg';
+const DEBUG = process.env['COMPILATION_MODE'] === 'dbg'
```
Comment (jbedard) [inline packages/rollup/install.md:127]: removing a semicolon?

Comment (alexeagle) [reply packages/rollup/install.md:127]: yeah, would you like me to put it back? just boiling things down to minimum

Comment (jbedard) [reply packages/rollup/install.md:127]: I figure a full line of code would normally include it 🤷

File: packages/rollup/install.md [hunk 2]
```diff
@@ -122,5 +124,33 @@ You could use this value to perform extra production-only optimizations.
...
+```
+
+and configure Rollup differently when `DEBUG` is `true`.
```
Comment (jbedard) [inline packages/rollup/install.md:130]: "is `true` vs `false`."
? 🤷

File: packages/rollup/rollup_bundle.bzl [hunk 3]
```diff
@@ -3,27 +3,22 @@
...
+See the <a href="https://rollupjs.org/guide/en/#command-line-flags">Rollup CLI docs</a> for a complete list of supported arguments.""",
         default = [],
     ),
     "config_file": attr.label(
-        doc = """A rollup.config.js file
+        doc = """A `rollup.config.js` file
 
-Passed to the --config 
-See https://rollupjs.org/guide/en/#configuration-files
+Passed to the `--config` option, see [the doc](https://rollupjs.org/guide/en/#configuration-files)
```
Comment (jbedard) [inline packages/rollup/rollup_bundle.bzl:21]: "see [the config doc]" or "see [the configuration doc]"?

**Review by jbedard — APPROVED** [review]
Minor comments but also fine as-is

**Review by mattem — APPROVED** [review]

**Pull request merged**